### ISSUE TITLE
Label gets trimmed 

### ISSUE THREAD

**Issue by ubaierbhat** [roughike/BottomBar#263]
What is the maximum length for a label? My labels are getting trimmed even when there is enough space around them. Is it possible to have a label in two lines when the icons are also visible?

**Comment by NickM-27** [comment]
Following the material design spec there should only be single line for the text of all items.

**Comment by ubaierbhat** [comment]
Yes. Forgot about that 👍  .. Can we do something about the trimming problem because I have plenty of space. I have tried to reduce the text size, which helps but I still loose few characters.
